src/poly/modern.js

```diff
@@ -1 +1,2 @@
 import './safari-nomodule'; // Safari 10.1 supports modules, but does not support the `nomodule` attribute - https://gist.github.com/samthor/64b114e4a4f539915a95b91ffd340acc
+import 'intersection-observer';
```

Comment:
As above - let's add a comment for why we need this

Reply:
I don't actually think we need to include this here - only in the legacy bundle - see https://github.com/bbc/simorgh/blob/latest/docs/JavaScript-Bundling-Strategy.mdx#why-create-modern-and-legacy-bundles.

We should be able to use `yarn build && yarn start` to verify that the view tracking works locally on Opera Mini 🤞